html/controls/mobile.js

```diff
@@ -15,7 +15,8 @@ $sb(function() {
 	setupJamControlPage();
 	setupPeriodTimePage();
 	setupTeamScorePage();
-	setupPenaltyTimePage();
+	//setupPenaltyTimePage();
```

Comment:
Why is this commented out?

Reply:
the UI side of the penalty Timing page is already commented out, wanted to prevent any of the js side of initializing as well.  I'm happy to remove the line instead of commenting out if you'd prefer.

Reply:
I'd prefer it removed, we can always get it back from source control.